packages/design-system-react/src/components/avatar-icon/README.mdx

```diff
@@ -0,0 +1,94 @@
+import { Controls, Canvas } from '@storybook/blocks';
+
+import * as AvatarIconStories from './AvatarIcon.stories';
+
+# AvatarIcon
+
+Avatar reserved for representing static icons inside of an avatar.
+
+> Note: For icon buttons, use the [ButtonIcon](/docs/react-components-buttonicon--docs) component.
```

Comment:
Adding not as per the [insight report ](https://www.figma.com/board/pQa2vI7r08pHiqsAPTWdQ8/Avatar%2C-Token%2C-Network%2C-Icon%2C-Badge-Audit-and-Insight-Report?node-id=1420-290&t=LQJI3xqByF8hdbgP-1)